data-model-translator-wrapper/src/main/java/ai/aitia/arrowhead/dmtpw/service/model/DataModelTranslationTask.java

```diff
@@ -0,0 +1,65 @@
+package ai.aitia.arrowhead.dmtpw.service.model;
+
+import java.util.UUID;
+
+import eu.arrowhead.dto.enums.DataModelTranslationTaskStatus;
+
+public class DataModelTranslationTask {
+
+	//=================================================================================================
+	// members
+
+	private UUID uuid;
+	private DataModelTranslationTaskStatus status;
+	private String resultMimeType;
+	private String payload;
+
+	//=================================================================================================
+	// methods
+
+	//-------------------------------------------------------------------------------------------------
+	public DataModelTranslationTask(final String resultMimeType, final String payload) {
+		this.uuid = UUID.randomUUID();
+		this.status = DataModelTranslationTaskStatus.PENDING;
+		this.resultMimeType = resultMimeType;
+		this.payload = payload;
+	}
+
+	//=================================================================================================
+	// boilerplate
+
+	//-------------------------------------------------------------------------------------------------
+	public UUID getUuid() {
+		return uuid;
+	}
+
+	//-------------------------------------------------------------------------------------------------
+	public DataModelTranslationTaskStatus getStatus() {
+		return status;
+	}
+
+	//-------------------------------------------------------------------------------------------------
+	public void setStatus(DataModelTranslationTaskStatus status) {
+		this.status = status;
+	}
+
+	//-------------------------------------------------------------------------------------------------
+	public String getResultMimeType() {
+		return resultMimeType;
+	}
+
+	//-------------------------------------------------------------------------------------------------
+	public void setInputModelId(final String resultMimeType) {
```

Comment:
I don't think this can be changed after creation.

Irrelevant, but the name is wrong too